athlinks-schema.ttl

```diff
@@ -195,81 +175,59 @@ athlinks:agePercentage
   rdfs:domain athlinks:IntervalResult;
   rdfs:range xsd:decimal .
 
-athlinks:isHidden
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isVisible
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isEnabled
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isApproved
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isHandicap
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:isNavigable
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
-
-athlinks:elevUnit
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:TerrainSpecifics;
-  rdfs:range xsd:float .
-
-athlinks:altitude
-  a owl:DataTypeProperty ;
-  rdfs:domain athlinks:TerrainSpecifics;
-  rdfs:range xsd:float .
-
-athlinks:startMarked
-  a owl:DataTypeProperty ;
+athlinks:elevation a owl:DataTypeProperty;
+  rdfs:label "Elevation";
+  rdfs:comment "Elevation";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitativeValue.
+
+athlinks:altitude a owl:DataTypeProperty;
+  rdfs:label "Altitude";
+  rdfs:comment "Altitude";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitativeValue.
+
+athlinks:elevationGradeType a owl:DataTypeProperty;
+  rdfs:label "Elevation Grade";
+  rdfs:comment "Elevation Grade";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitiveValue.
+
+athlinks:elevationType a owl:DataTypeProperty;
+  rdfs:label "Elevation";
+  rdfs:comment "Elevation";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitiveValue.
+
+athlinks:altitudeType a owl:DataTypeProperty;
+  rdfs:label "Altitude Type";
+  rdfs:comment "Altitude Type";
+  rdfs:domain schema:Landform;
+  rdfs:range schema:QuantitiveValue.
+
+athlinks:distanceUnit a owl:DataTypeProperty;
+  rdfs:label "Default Distance Unit";
+  rdfs:comment "Default Distance Unit";
   rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
+  rdfs:range schema:QuantitiveValue.
 
-athlinks:finishMarked
-  a owl:DataTypeProperty ;
+athlinks:distanceDisplayUnit a owl:DataTypeProperty;
+  rdfs:label "Distance Unit to Display";
+  rdfs:comment "Distance Unit to Display";
   rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:boolean .
+  rdfs:range schema:QuantitiveValue.
 
-athlinks:distanceUnit
-  a owl:DataTypeProperty ;
+athlinks:createUser a owl:DataTypeProperty;
+  rdfs:label "ID of User Who Created Race";
+  rdfs:comment "ID of User Who Created Race";
   rdfs:domain athlinks:AthleticRace;
-  rdfs:range xsd:float .
+  rdfs:range xsd:int.
```

Comment:
This should at least have a range of Person or Organization. The name should likely be createdBy..And at that point.. We may be better off just using: http://schema.org/creator.